components/Video.ts

```diff
@@ -0,0 +1,30 @@
+import { jwt } from 'twilio';
+import { connect } from 'twilio-video';
+
+const getRandomIntInclusive = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+const generateToken = () => {
+  const twilioAccountSid = process.env.TWILIO_SID;
+  const twilioApiKey = process.env.TWILIO_API_KEY;
+  const twilioApiSecret = process.env.TWILIO_API_SECRET;
+  const token = new jwt.AccessToken(
+    twilioAccountSid,
+    twilioApiKey,
+    twilioApiSecret
+  );
+  // @ts-ignore
+  token.identity = `somebody-${getRandomIntInclusive(1, 100)}`;
```

Comment:
You could use `Date.now()`(returns milliseconds since epoch) here to make it a little harder to have a collision!

Reply:
Ah, I like having something human-distinguishable at a glance (45 vs 79, rather than 1584936034273 vs 1584936045323).

I think in the future we should just ask people for their name.